cypress/integration/basic/person_modal.spec.js

```diff
@@ -0,0 +1,28 @@
+describe('Test that the person show modal can be opened', () => {
```

Comment:
Update the describe() and it() language as described at https://github.com/dssit/roles-management/pull/349